### ISSUE TITLE
Request: perhaps a tool to use Windows Sandbox for running MABS

### ISSUE THREAD

**Issue by hydra3333** [m-ab-s/media-autobuild_suite#2471]
Hello. Some people may not want to have MABS install stuff, and prefer to run it in a VM such as a (native) Windows Sandbox and, save just the resulting files and the rest disappears when the Windows Sandbox is closed. Gooder security.

Here's a start at something like that, perhaps someone may care to consider broadening it to be more generally useful ?
(It's only useful to me with my specific folder structure and whatnot).

https://github.com/m-ab-s/media-autobuild_suite

Cheers

**Comment by 1480c1** [comment]
After messing around with Windows Sandbox, I do not see any easy way to script a way to spin one up and add files to it. Perhaps something might be doable with a normal VM, but that will require more knowledge of how to interact with hyper-v from powershell that I do not have at this time, so nothing soon.

**Comment by hydra3333** [comment]
OK. Thanks for looking.

PS after a Sandbox is spun up by the .wsb the "auto-executed" associated Setup.bat runs which in turn creates a new .bat file on the Sandbox's desktop which the user is instructed to run ... that does a heap of stuff including downloads etc.    
I found it easier to do things that way, inside the Sandbox than to try to poke stuff in from outside the sandbox ... although I did try keeping stuff in a shared folder which both the host and the sandbox could see but which was only executed within the sandbox.
